# Representation of a vehicle state in the SML World

uint32 vehicle_id
string class_name

float64 x
float64 y

float64 yaw
